pydecred/rpc.py

```diff
@@ -290,6 +447,446 @@ def get(k, obj):
     return obj[k] if k in obj else None
 
 
+class GetWorkResult:
+    """getworkresult"""
+
+    def __init__(
+        self, data, target,
+    ):
+        """
+        Args:
+            data (str): Hex-encoded block data.
+            target (str): Hex-encoded little-endian hash target.
+        """
+        self.data = data
+        self.target = target
+
+    @staticmethod
+    def parse(obj):
+        """
+        Parse the GetWorkResult from the decoded RPC response.
+
+        Args:
+            obj (object): The decoded dcrd RPC response.
+
+        Returns:
+            GetWorkResult: The GetWorkResult.
+        """
+        return GetWorkResult(data=obj["data"], target=obj["target"],)
+
+
+class GetTxOutResult:
+    """gettxoutresult"""
+
+    def __init__(
+        self, bestBlock, confirmations, value, scriptPubKey, version, coinbase,
+    ):
+        """
+        Args:
+            bestBlock (str): The block hash that contains the transaction
+                output
+            confirmations (int): The number of confirmations
+            value (float): The transaction amount in DCR
+            scriptPubKey (ScriptPubKeyResult): The public key script used to pay
+                coins as a JSON object
+            version (int): The transaction version
+            coinbase (bool): Whether or not the transaction is a coinbase
+        """
+        self.bestBlock = bestBlock
+        self.confirmations = confirmations
+        self.value = value
+        self.scriptPubKey = scriptPubKey
+        self.version = version
+        self.coinbase = coinbase
+
+    @staticmethod
+    def parse(obj):
+        """
+        Parse the GetTxOutResult from the decoded RPC response.
+
+        Args:
+            obj (object): The decoded dcrd RPC response.
+
+        Returns:
+            GetTxOutResult: The GetTxOutResult.
+        """
+        return GetTxOutResult(
+            bestBlock=obj["bestblock"],
+            confirmations=obj["confirmations"],
+            value=obj["value"],
+            scriptPubKey=ScriptPubKeyResult.parse(obj["scriptPubKey"]),
+            version=obj["version"],
+            coinbase=obj["coinbase"],
+        )
+
+
+class Choice:
+    """
+    Choice models an individual choice inside an Agenda.
+    """
+
+    def __init__(
+        self, ID, description, bits, isAbstain, isNo, count, progress,
+    ):
+        """
+        Args:
+            ID (str): Unique identifier of this choice.
+            description (str): Description of this choice.
+            bits (int): Bits that identify this choice.
+            isAbstain (bool): This choice is to abstain from change.
```

Comment:
Does abstain have the same weight as no?

Reply:
In practice, they are never set together. We could potentially not read them at all, and use `@property`  for getters based on the `choiceID`, but this is good for now.